packages/cli/src/databases/repositories/test-definition.repository.ee.ts

```diff
@@ -0,0 +1,60 @@
+import type { FindManyOptions, FindOptionsWhere } from '@n8n/typeorm';
+import { DataSource, In, Repository } from '@n8n/typeorm';
+import { Service } from 'typedi';
+
+import { TestDefinition } from '@/databases/entities/test-definition.ee';
+import type { ListQuery } from '@/requests';
+
+@Service()
+export class TestDefinitionRepository extends Repository<TestDefinition> {
+	constructor(dataSource: DataSource) {
+		super(TestDefinition, dataSource.manager);
+	}
+
+	async getMany(sharedWorkflowIds: string[], options?: ListQuery.Options) {
```

Comment:
Why use `sharedWorkflowIds`? what is "shared" about these?

Reply:
This term is widely used in other parts of the code where we need to check if user has access to the workflow. I guess because it was introduces with the workflow sharing feature. `accessibleWorkflowIds` would be better here

Reply:
I see. Yes, I like your suggestion better, but if it's a common idiom, then let's stick to what people know :)